core.cc

```diff
@@ -90,10 +93,12 @@ NAN_METHOD(SysLog) {
 
     if (message) {
         Nan::AsyncQueueWorker(new Worker(callback, priority, message));
-    } else if(!callback->IsEmpty()) {
```

Comment:
if `info[2]->IsFunction()` is false, then there is no callback... but I have to pass something into the worker ctor, and then its default HandleOKCallback() will try to use that something. So, I think I need my own implementation... unless its possible to create a no-op Nan::Callback()? Maybe `else callback = new Nan::Callback()`? ... but I just tried that, it compiles, but segvs.